### ISSUE TITLE
Issue with Kernel 4.4.103-36 and DKMS 2.2.1.0

### ISSUE THREAD

**Issue by jessecooper** [DisplayLink/evdi#110]
Followed the advice of [this comment](https://github.com/DisplayLink/evdi/issues/24#issuecomment-217211038) to get the module to compile in the displaylink-install. Here is a diff:
```
--- ../evdi_fb.c        2017-10-06 05:58:54.000000000 -0500
+++ evdi_fb.c   2017-12-18 14:36:25.458211744 -0600
@@ -229,7 +229,7 @@
                ret =
                    dma_buf_begin_cpu_access(
                        ufb->obj->base.import_attach->dmabuf,
-#if KERNEL_VERSION(4, 6, 0) > LINUX_VERSION_CODE
+#if KERNEL_VERSION(4, 6, 0) < LINUX_VERSION_CODE
                        0, ufb->obj->base.size,
 #endif
                        DMA_FROM_DEVICE);
@@ -247,7 +247,7 @@
 
        if (ufb->obj->base.import_attach)
                dma_buf_end_cpu_access(ufb->obj->base.import_attach->dmabuf,
-#if KERNEL_VERSION(4, 6, 0) > LINUX_VERSION_CODE
+#if KERNEL_VERSION(4, 6, 0) < LINUX_VERSION_CODE
                                       0, ufb->obj->base.size,
 #endif
```

Hopefully this will help if anyone else runs into the same issue. A better solution might be to just change the KERNEL_VERSION(4, 6, 0) to KERNEL_VERSION(4, 4, 0).

**Comment by xmentos** [comment]
Because of recent LTS kernel patches I think it would be more correct to change the macros to the following:

`#if KERNEL_VERSION(4, 4, 102) > LINUX_VERSION_CODE`

**Comment by displaylink-dkurek** [comment]
Some time has passed. I hope we have solved already all problems with 4.4 compilation :) Will close this issue.
